src/components/qualityOptions.js

```diff
@@ -5,6 +5,13 @@ define(['globalize'], function (globalize) {
 
         var maxStreamingBitrate = options.currentMaxBitrate;
         var videoWidth = options.videoWidth;
+        var videoHeight = options.videoHeight;
+
+        //If the aspect ratio is less than 16/9 (1.77), set the height as if it were pillerboxed.
```

Comment:
```suggestion
        //If the aspect ratio is less than 16/9 (1.77), set the width as if it were pillarboxed.
```

You're setting the with, not the height :)